handbook/engineering/distribution/goals.md

```diff
@@ -64,17 +64,17 @@ We will start by looking at our Kubernetes deployment and working on an easier u
   - Design customization workflow.
     - Potentially Kustomize would still be used for last-mile changes and non-standard derivations.
 
-### Improve the debugging and troubleshooting process
+### Further standardize the debugging and troubleshooting process
 
-As we deploy Sourcegraph to multiple different environments, we need to provide a consistent and straightforward process to debug issues. We are currently lacking tools to collect debugging information (configuration, type, size, diff from upstream, etc) consistently and a process to capture the output of debugging sessions to feed back into our priorities and documentation.
-We will initially focus on reducing the time it takes to collect troubleshooting information.
+As we deploy Sourcegraph to multiple different environments, we need to provide a consistent and straightforward process to debug issues across our deployments and customers' deployments. We have some standardized tooling such as Grafana dashboards, alerting, and Jaeger tracing - but we need to further improve the usage of these and further standardized/formalize the debugging process.
 
 - **Owner**: TBD
 - **Status**: Not started. Unknown amount of work.
 - **Outcomes**:
   - We can categorize and capture the amount of effort spent on different incident types.
-  - We can provide a straightforward set of tools to collect initial debugging and deployment information.
-  - TBD
+  - We provide a straightforward set of processes/tools to collect initial debugging and deployment information (configuration, type, size, diff from upstream, recently-firing alerts, logs, etc.)
+  - Alerts in services owned by other teams at Sourcegraph are well-defined, extensive, and have excellent documentation site admins can understand.
+  - The tooling we rely on to debug our own infrastructure is aligned with the tooling we use to debug customer deployments.
 - **Milestones**: TBD
 
 ### [Split Cloud infrastructure into separate GCP projects](https://github.com/orgs/sourcegraph/projects/92)
```

Comment:
What is the status of this? It seems almost completely done except some docs?

Also, can we create a "past goals" section in this doc (or in a `past_goals.md` file) which includes all the goals we successfully did thus far? I think that is a great resource for looking historically, and for sharing with candidates when we start hiring more folks.

Check out https://github.com/sourcegraph/about/pull/1104/files#r519076983 which includes my review of the roadmap I put out earlier this year - it's really cool to see in retrospect how many of these points we did make a lot of progress on. We should find a way to communicate this (internally and to the rest of the org)

Reply:
I created another PR to add a past_goals.md doc - I think it's super interesting! https://github.com/sourcegraph/about/pull/1974

Reply:
The status is reflected in the goal, its partially missing `Document project and folder usage guidelines.` and filling this outcome `We can set spending limits for dynamic environments.`. I dont think it should be moved yet, unless we choose to de-scope those.

Reply:
Got it, `We can set spending limits for dynamic environments.` makes it more clear to me there is more than just "follow up on docs" here, any objection to me adding that to "milestones"?

Reply:
https://github.com/sourcegraph/about/pull/1973/files#r520010960